BUILD.bazel

```diff
@@ -208,6 +208,9 @@ GOLDEN_UPDATING_UNIT_TESTS = [
     "com.google.api.generator.gapic.composer.rest.ServiceClientTestClassComposerTest",
     "com.google.api.generator.gapic.composer.rest.ServiceSettingsClassComposerTest",
     "com.google.api.generator.gapic.composer.rest.ServiceStubSettingsClassComposerTest",
+    "com.google.api.generator.spring.composer.SpringAutoConfigClassComposerTest",
```

Comment:
Can you try to use Maven command like this `mvn test -DupdateUnitGoldens -Dtest=JavaCodeGeneratorTest` to update these tests?  [DEVELOPMENT.md](https://github.com/googleapis/gapic-generator-java/blob/main/DEVELOPMENT.md) has more details about it. Ideally we don't want to touch anything in Bazel.

Reply:
Ah I hadn't realized this alternative - thank you! Just made the switch to not modify this file.

Reply:
Yeah, the Maven build in gapic-generator-java is not completely done, so some of the functionality may not be completely working and we never publicized it. Did you confirm that you can use the maven command to update golden test files?

Reply:
Yup, I checked locally that the golden files were able to be generated using the maven command instead.

Reply:
Perfect, thanks!